lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -4,4 +4,77 @@
             data-user-create-subcomment="<%= canCreateSubComment %>">
         <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
     </div>
+
+    <div class="team-detail">
+        <h4><%- gettext("Team Details") %></h4>
+        <% if (isMember) { %>
+            <div class="team-member">
+                <span><%- gettext("You are a member of this team.") %></span>
+            </div>
+        <% } %>
+        <div class="team-members">
+            <span class="sr"><%- gettext("Team members profile") %></span>
+            <div class="members-info"></div>
+        </div>
+
+        <div class="team-capacity">
+            <span class="sr"><%- gettext("Team capacity") %></span>
+            <span><%- membershipText %></span>
+        </div>
+
+        <div class="team-country">
+            <span class="sr"><%- gettext("Team location") %></span>
+            <i class="u-field-icon icon fa fa-globe fa-fw" aria-hidden="true"></i>
+            <span>
+                <%=
+                    interpolate_text(
+                        gettext("{country}"),
+                        {
+                            'country': country,
+                        }
+                    )
+                %>
+            </span>
+        </div>
+        <div class="team-language">
+            <span class="sr"><%- gettext("Team language") %></span>
+            <i class="icon fa fa-comment-o fa-fw" aria-hidden="true"></i>
+            <span>
+                <%=
+                    interpolate_text(
+                        gettext("{language}"),
+                        {
+                            'language': language,
+                        }
+                    )
+                %>
+            </span>
+        </div>
+
+        <% if (isMember) { %>
+            <div class="leave-team">
+                <a href="#"><%- gettext("Leave Team") %></a>
+                <span class="sr"><%- gettext("Click this to leave the team") %></span>
+            </div>
+
+            <div class="divider-lv1"></div>
+
+            <div class="invite-team">
+                <h4 class="invites-header"><%- gettext("Invite Others") %></h4>
+                <% if (hasCapacity) { %>
+                    <span class="sr"><%- gettext("Copy this url to invite others") %></span>
+                    <input type="text" class="invite-link-input" value="<%= inviteLink %>" readonly>
```

Comment:
I like the idea of this screen reader only text, but something tells me its unnecessary.  It's very close to the same message that is displayed visually.  The same text could be used.  What we need to do to actually make this effective is to programmatically associate it with the input.    Put an ID attribute on the "invite-text" below and then add aria-describedby="[IDref]" on the ```<input>``` that way when a screen reader user moves focus to the control, they will get the associated help text.  otherwise, they will never hear it.